MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm

```diff
@@ -426,14 +481,16 @@
 								  ? VK_SUBPASS_CONTENTS_SECONDARY_COMMAND_BUFFERS
 								  : VK_SUBPASS_CONTENTS_INLINE);
 
+	// Track both rendering and resolve attachments, alternating a rendering attachment, then it's corresponding resolve attachment.
 	uint32_t maxAttCnt = (pRenderingInfo->colorAttachmentCount + 2) * 2;
 	MVKImageView* imageViews[maxAttCnt];
 	VkClearValue clearValues[maxAttCnt];
 
+	// Assemble the list of attachments. This must be done identically to how the dynamic renderpass assembles attachments.
```

Comment:
That's fragile. There must be a way to keep SPOT without sacrificing performance or turning the code into an eldritch horror... well, any more than it already is.

There should at least be a similar comment in the dynamic renderpass code so future contributors will remember to update both.